.gitignore

```diff
@@ -14,10 +20,19 @@ cookbooks/sysctl/
 cookbooks/ubuntu/
 cookbooks/yum/
 cookbooks/yum-epel/
-environments/*
+cookbooks/7-zip/
```

Comment:
Is it possible to do the following instead of calling out pulled down cookbooks:
````
cookbooks/*/
!cookbooks/bcpc
!cookbooks/bcpc-hadoop
!cookbooks/hannibal
[...]
````